src/main/java/com/internship/backend/model/Users.java

```diff
@@ -19,12 +21,9 @@
 @Setter
 @Getter
 @Valid
+@Builder
 public class Users {
```

Comment:
Could this class name be renamed to user, since it holds info for a singe user (username, password, ....) ?